apps/site/src/components/pages/wordpress/header.tsx

```diff
@@ -0,0 +1,149 @@
+          sx={{
+            textTransform: "uppercase",
+            color: ({ palette }) => palette.purple[700],
+            mb: 2.25,
+            fontWeight: 500,
+            letterSpacing: "0.06rem",
+            mx: "auto",
+          }}
+          textAlign="center"
+          variant="bpSmallCaps"
+        >
+          Early access
+        </Typography>
+
+        <Box
+          sx={({ breakpoints }) => ({
+            display: "flex",
+            flexDirection: "row",
+            justifyContent: "center",
+            alignItems: "center",
+            mb: 2.25,
+            [breakpoints.down("md")]: {
+              flexDirection: "column",
+            },
+            fontSize: "clamp(3rem, 5vw, 7rem)",
+          })}
+        >
+          <Box
+            sx={({ breakpoints }) => ({
+              display: "flex",
+              flexDirection: "row",
+              alignItems: "center",
+              mr: 2.25,
+              mb: 1,
+              [breakpoints.down("md")]: {
+                mr: 0,
+              },
+            })}
+          >
+            <BpWpIcons />
+          </Box>
+
+          <Typography
+            variant="bpHeading1"
+            textAlign="center"
+            sx={{
+              lineHeight: 1,
+              color: ({ palette }) => palette.gray[90],
+              fontSize: "1em",
+            }}
+          >
+            WordPress Plugin
+          </Typography>
+        </Box>
+
+        <Typography
+          variant="bpBodyCopy"
+          lineHeight={1.4}
+          color={theme.palette.gray[80]}
+          textAlign="center"
+          sx={{
+            margin: "0 auto",
+            fontSize: { xs: "1.25rem", md: "1.45rem" },
+          }}
+        >
+          Block Protocol blocks work across all Þ supporting applications
+        </Typography>
+        <Typography
+          variant="bpBodyCopy"
+          lineHeight={1.4}
+          color={theme.palette.gray[80]}
+          textAlign="center"
+          sx={{
+            margin: "0 auto",
+            fontWeight: 700,
+            fontSize: { xs: "1.25rem", md: "1.45rem" },
+          }}
+        >
+          ...now including WordPress, powering 43% of the web
+        </Typography>
+
+        <Box
+          sx={{
+            marginTop: 6.25,
+            maxWidth: 500,
+            display: "flex",
+            justifyContent: "center",
+          }}
+        >
+          <EarlyAccessCTA />
+        </Box>
+      </Container>
+
+      <Box sx={{ width: "100vw", position: "relative" }}>
+        <Image layout="responsive" src={wpHelixImage} />
+        <Box
+          sx={{
+            position: "absolute",
+            top: 0,
+            left: "50%",
+            transform: "translateX(-50%)",
+            maxWidth: { xs: "95%", md: "75%", lg: "60%" },
+            width: "100%",
+            div: {
+              width: "100%",
+              height: "100%",
+            },
+          }}
+        >
+          <Stream
```

Comment:
@luisbettencourt **Blocking merge:** please can we add a placeholder/loading state while the Cloudflare Stream video player loads in?

Reply:
Added in 10ff8f0c5cfc10fa3da6a2baa07d953b1c6c4200.

![skeleton](https://user-images.githubusercontent.com/37453800/208083171-7c9f429e-c99b-42d8-ab94-b0dad3069cdc.gif)